addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_ubl_nl.py

```diff
@@ -75,6 +78,14 @@ def setUpClass(cls,
             'country_id': cls.env.ref('base.nl').id,
         })
 
+        cls.tax_10_fixed = cls.env['account.tax'].create({
+            'name': 'Test Tax',
+            'amount_type': 'fixed',
+            'include_base_amount': True,
+            'country_id': cls.env.ref('base.nl').id,
+            'amount': 10.0,
```

Comment:
set the sequence as '2' for instance, and in the creation of the 7% tax, set the sequence as '1'.

Reply:
> set the sequence as '2' for instance, and in the creation of the 7% tax, set the sequence as '1'.

Sorry- maybe I am misunderstanding, but it should be the other way around right? The 7% tax should be sequence=2 and the 10 unit fixed tax should be sequence=1 to get the desired `(50 + 10) * 1.07`?

Reply:
oooops yep, that's right :)